live-editing/configs/TreeGridConfigGenerator.ts

```diff
@@ -387,6 +390,17 @@ export class TreeGridConfigGenerator implements IConfigGenerator {
             shortenComponentPathBy: "/tree-grid/"
         }));
 
+        configs.push(new Config({
+            additionalFiles: ["/src/app/tree-grid/data/foods.ts"],
+            appModuleConfig: new AppModuleConfig({
+                imports: [IgxTreeGridModule, TreeGridExcelStyleFilteringStyleComponent, IgxInputGroupModule],
```

Comment:
Do we need the **IgxInputGroupModule** here?

Reply:
Addressed.